ingestion/src/metadata/ingestion/source/storage/s3/metadata.py

```diff
@@ -316,31 +296,28 @@ def _get_sample_file_path(
         or None in the case of a non-structured metadata entry, or if no such keys can be found
         """
         prefix = self._get_sample_file_prefix(metadata_entry=metadata_entry)
-        # no objects found in the data path
-        if not self.prefix_exits(bucket_name=bucket_name, prefix=prefix):
-            logger.warning(f"Ignoring metadata entry {prefix} - no files found")
-            return None
```

Comment:
any reason we are removing this check? OK to update the connector to unify how we load the manifest file, but the rest of the logic I believe was ok

Reply:
<img width="955" alt="Screenshot 2023-09-08 at 1 47 56 PM" src="https://github.com/open-metadata/OpenMetadata/assets/40225091/9f6ebaa0-9eba-4186-b6f0-b2a1eb487ec0">
prefix exists does the same thing, if prefix doesn't exists, it returns None, we are doing the same thing here, just reducing repitition